src/app/app.module.ts

```diff
@@ -54,6 +54,7 @@ import {SharedModule} from './shared/shared.module';
 import {SubmissionListComponent} from './submission-list/submission-list.component';
 import {TemplateQuestionnaireModule} from './template-questionnaire/template-questionnaire.module';
 import {WelcomeComponent} from './welcome/welcome.component';
+import {GeoAccessionDownloadComponent} from './geo-accession-download/geo-accession-download.component';
```

Comment:
I'm really nitpicking now but I will not reject this PR because of this. If you use "optimize imports" (ctrl+shift+o) in WebStorm, it will alphabetize everything and keep a structured format

Reply:
these suggestions are always good and shouldn't be seen as nitpicking :)

Reply:
Nitpicks with PR approval are good. 😛